src/main/java/core/basesyntax/service/impl/PurchaseAction.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class PurchaseAction implements ActionHandler {
+    @Override
+    public void apply(String fruit, int quantity) {
+        Storage.removeFruit(fruit, quantity);
+    }
```

Comment:
Handling Purchase operation.
Check result balance before saving it in the Storage - it should be positive. Throw `RuntimeException` in case the balance is negative.
Currently, there is no check for the result balance to ensure it is positive after the purchase. You should implement this check to avoid negative balances in the storage.